<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>683</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{

}

QFrame{
	
	border: 0px none;

}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Automated Visual Inspection of Silicon Detectors in CMS Experiment&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="vesitLogo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/vesit.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tifrLogo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/tifr-logo.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>350</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>350</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>10</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="usernameInput">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: black; border: 2px solid black;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="passwordLabel">
              <property name="styleSheet">
               <string notr="true">font: 75 11pt &quot;Vrinda&quot;;</string>
              </property>
              <property name="text">
               <string>Password:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="PasswordEdit" name="passwordInput">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: black; border: 2px solid black;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>15</number>
              </property>
              <item>
               <widget class="LoginButton" name="loginBtn">
                <property name="minimumSize">
                 <size>
                  <width>182</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="accessibleName">
                 <string>loginBtn</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
color: white;
border-style: none;
border-width: 3px;
border-radius: 8px;
border-color: #0F49B5;
border-bottom: 3px solid rgb(58, 143, 186);
font: bold 14px;
min-width: 10em;
padding: 6px;
	background-color: rgb(31, 87, 115);
}

QPushButton:hover{
background-color: rgb(58, 143, 186)
}</string>
                </property>
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="ForgotPassButton" name="resetPasswordBtn">
                <property name="styleSheet">
                 <string notr="true">QPushButton{
color: white;
border-style: none;
border-width: 3px;
border-radius: 8px;
border-color: #0F49B5;
border-bottom: 3px solid rgb(58, 143, 186);
font: bold 14px;
min-width: 10em;
padding: 6px;
	background-color: rgb(191, 21, 21);
}

QPushButton:hover{
background-color: rgb(58, 143, 186)
}</string>
                </property>
                <property name="text">
                 <string>Reset Password</string>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="RegisterButton" name="registerBtn">
                <property name="styleSheet">
                 <string notr="true">QPushButton{
color: white;
border-style: none;
border-width: 3px;
border-radius: 8px;
border-color: #0F49B5;
border-bottom: 3px solid rgb(58, 143, 186);
font: bold 14px;
min-width: 10em;
padding: 6px;
	background-color: rgb(12, 135, 75);
}

QPushButton:hover{
background-color: rgb(58, 143, 186)
}</string>
                </property>
                <property name="text">
                 <string>Register</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="usernameLabel">
              <property name="styleSheet">
               <string notr="true">font: 75 11pt &quot;Vrinda&quot;;</string>
              </property>
              <property name="text">
               <string>Username:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">qtwidgets</header>
  </customwidget>
  <customwidget>
   <class>LoginButton</class>
   <extends>QPushButton</extends>
   <header location="global">src.buttons.LoginButton</header>
  </customwidget>
  <customwidget>
   <class>ForgotPassButton</class>
   <extends>QPushButton</extends>
   <header location="global">src.buttons.ForgotPassButton</header>
  </customwidget>
  <customwidget>
   <class>RegisterButton</class>
   <extends>QPushButton</extends>
   <header location="global">src.buttons.RegisterButton</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
